neore/scripts/colors_shell.sh

```diff
@@ -33,9 +33,9 @@ _message(){
    isOk=$(echo "$1"|grep -i "\\[OK\\]")
    isUser=$(echo "$1"|grep -i "\\[USER\\]")
    isDash=`echo -en|grep "\-en"`
-   outLog="/dev/stdout"
+   outLog="/dev/tty"
```

Comment:
I guess this is something you detected wrong in the releasing sofware, right? Thanks! Nevertheless, I would avoid mixing things and I would focus this PR in the PostgreSQL Sink, detailing the bug in an issue and moving the fixing to future PRs.